test/integration/merkle-lockup/ll/clawback/clawback.t.sol

```diff
@@ -8,6 +8,9 @@ import { Errors } from "src/libraries/Errors.sol";
 import { MerkleLockup_Integration_Test } from "../../MerkleLockup.t.sol";
 
 contract Clawback_Integration_Test is MerkleLockup_Integration_Test {
+    uint40 internal firstClaimTime;
+    uint256 internal claimIndex = 1;
```

Comment:
we don't need this variable, we can simply not make the claim in `givenCampaignExpired` since we have now `afterFirstClaim`

Reply:
Right. You are right. We dont need this.